langstream-cli/src/main/java/ai/langstream/cli/commands/docker/LocalRunApplicationCmd.java

```diff
@@ -241,6 +249,17 @@ public void run() {
 
         updateLocalDockerRunProfile(tenant);
 
+        Runtime.getRuntime()
+                .addShutdownHook(
+                        new Thread(
+                                () -> {
+                                    log("Cleaning environment");
```

Comment:
can we log this only if there are "temporary files"

Reply:
done